### PR TITLE
feat(sms): Allow sending SMS messages to Romania.

### LINKED ISSUE #4829: Allow SMS testing in Romania
Softvision needs to be able to test the send SMS feature from Romania. Make this a possibility.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Allow sending SMS messages to Romania. Instead of using a click handler on
the .success element, allow testers to open /verify_email with a `?country=<xx>`
query parameter where country is one of `CA`, `GB`, `RO`, `US`.

fixes #4829 

@philbooth, @vladikoff, @vbudhram - r?

### CODE REVIEW

File: app/scripts/views/sms_send.js [hunk 1]
```diff
@@ -93,10 +52,24 @@ define(function (require, exports, module) {
...
+        // /sms/status endpoint
+        const country = this.relier.get('country');
+        const prefix = CountryTelephoneInfo[country].prefix;
+        if (prefix !== CountryTelephoneInfo.US.prefix) {
+          phoneNumber = prefix;
+        }
+      }
 
       return {
-        country,
```
Comment (shane-tomlinson) [inline app/scripts/views/sms_send.js]: This needs to be added back in, the `tel-input` makes use of it!

File: app/scripts/lib/country-telephone-info.js [hunk 2]
```diff
@@ -48,14 +48,33 @@
...
+         // as well as an extra 0 before the 7 prefix.
+         const prefix = /^(\+40)?0?/;
+         if (prefix.test(num)) {
+           num = num.replace(prefix, '');
+         }
+         return `+40${num}`;
+       },
+       // +407xxxxxxxx, allow leading 0 for sloppiness.
+       pattern: /^(?:\+40)?0?7\d{8,8}$/,
+       prefix: '+40'
```
Comment (vbudhram) [inline app/scripts/lib/country-telephone-info.js]: Should you prefill with `+407`, quick googling seems to imply that this there mobile number [prefix](https://www.howtocallabroad.com/romania/).

Comment (shane-tomlinson) [inline app/scripts/lib/country-telephone-info.js]: Done

File: docs/query-params.md [hunk 3]
```diff
@@ -117,6 +117,15 @@ When the relier supports being linked to its `redirect_uri` without extra OAuth
 * /signup
 * /force_auth
 
+### `country`
+Force a country to be used when testing the SMS feature.
```
Comment (vladikoff) [inline docs/query-params.md]: 👍

File: tests/functional/send_sms.js [hunk 4]
```diff
@@ -62,6 +62,46 @@
...
+           }
+         }))
+         .then(testElementValueEquals(SELECTOR_SEND_SMS_PHONE_NUMBER, ''));
+     },
+
+     'with `country=RO`': function () {
+       return this.remote
+         .then(openPage(SEND_SMS_URL, SELECTOR_SEND_SMS_HEADER, {
+           query: {
+             country: 'RO'
```
Comment (vladikoff) [inline tests/functional/send_sms.js]: 👍

**Pull request merged**